source/security/enterprise-auth.txt

```diff
@@ -422,6 +422,25 @@ see the corresponding syntax.
                  .credential(credential)
                  .build());
 
+Kubernetes
+++++++++++
+
+If your application runs on a Kubernetes cluster, you can authenticate to MongoDB by using
+the {+driver-short+}'s built-in Kubernetes support.
+
+To specify Kubernetes OIDC as the authentication mechanism, set the following 
+options in your connection string:
+
+- ``authMechanism``: Set to ``MONGODB-OIDC``.
+- ``authMechanismProperties``: Set to ``ENVIRONMENT:k8s``. 
```

Comment:
We don't say this in the other sections - for example, under Azure IMDS connection string, we do not say set to `MONGODB-OIDC` and `ENVIRONMENT:azure`. Should we include the wording in all the other sections, or say less here?